app/Jobs/DatabaseBackupJob.php

```
@@ -131,8 +159,19 @@ public function handle(): void
+                    $serviceUuid = $this->database->service->uuid;
+                    $serviceName = str($this->database->service->name)->slug();
+                } else {
+                    if ($this->database->application_id) {
+                        $serviceUuid = generateApplicationContainerName($this->database->application);
+                        $serviceName = str($this->database->application->name)->slug();
+                    } else {
+                        $preview = $this->database->application_preview;
+                        $serviceUuid = generateApplicationContainerName($preview->application, $preview->pull_request_id);
+                        $serviceName = str($preview->application->name)->slug();
+                    }
+                }
```

PHPMD (2.15.0)

[error] The method handle uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)